app/presenters/formats/place_presenter.rb

```diff
@@ -0,0 +1,40 @@
+module Formats
+  class PlacePresenter < EditionFormatPresenter
+  private
+
+    def schema_name
+      'place'
+    end
+
+    def document_type
+      'place'
+    end
+
+    def details
+      details = { external_related_links: external_related_links }
+
+      details = details.merge(govspeak_fields)
+
+      if edition.place_type
+        details = details.merge(place_type: edition.place_type)
```

Comment:
The `merge` method creates a new hash, copies the combined contents into it, then we're assigning that new hash to the same local variable. 

Seems it might make more sense to just modify the existing hash in-place.

`details[:place_type] = edition.place_type`